man/man1/registercloudguest.1

```diff
@@ -45,6 +60,14 @@ Force a new registration. This causes the previous registration data to be
 discarded and the system to get re-registered as if it had never been
 registered previously. This is an "emergency" switch and should be used only
 in exceptional circumstances. This option and "--clean" are mutually exclusive.
+This option does not work for BYOS instances that use the update
```

Comment:
Since it is one sentence I think one mention of "BYOS instances" can be omitted : 

` This option does not work for BYOS instances that use the update infrastructure and will result in a registration failure.`

or 

`This option does not work and will result in a registration failure if used with BYOS instances.`